### PR TITLE
openvswitch: Ensure a consistent MTU size compatible with GRE tunnels

### PR DESCRIPTION
**Pull request by okurz**
So far common multi-machine tests set an MTU size but not always consistently and we want to ensure we have a properly working setup without relying on test maintainers to know the setup of machines.

Related progress issue: https://progress.opensuse.org/issues/151310

### CODE REVIEW

**Review by asdil12 — CHANGES_REQUESTED** [review]

File: script/os-autoinst-setup-multi-machine
Comment (Martchus) [reply script/os-autoinst-setup-multi-machine:72]: We just tried with 1450 and it might have helped. I suppose we should wait until we have gathered more experience until we merge this PR.

Comment (okurz) [reply script/os-autoinst-setup-multi-machine:72]: @nicksinger can you help me there?

Comment (sqozz) [reply script/os-autoinst-setup-multi-machine:72]: It was honestly just a blind guess based on what is written on https://docs.openvswitch.org/en/latest/faq/issues/:

> Sometimes this behavior is not desirable, for example with tunnels. The MTU of an internal interface can be explicitly set using the following command:
> 
> $ ovs-vsctl set int br0 mtu_request=1450

maybe we could use that as reference. OTOH we have it mentioned in the issue anyway.

Comment (nicksinger) [reply script/os-autoinst-setup-multi-machine:72]: It was honestly just a blind guess based on what is written on https://docs.openvswitch.org/en/latest/faq/issues/:

> Sometimes this behavior is not desirable, for example with tunnels. The MTU of an internal interface can be explicitly set using the following command:
> 
> $ ovs-vsctl set int br0 mtu_request=1450

maybe we could use that as reference. OTOH we have it mentioned in the issue anyway.

**Review by mimi1vx — APPROVED** [review]

**Pull request merged**
